projects/instant-ui-library/src/lib/button/models/button.model.ts

```diff
@@ -0,0 +1 @@
+export type ButtonType = 'primary' | 'secondary' | 'tertiary';
```

Comment:
Is the overlap between this and `color.model` wanted?

Reply:
This is not needed anymore.